scenes/quests/story_quests/equipo2/equipo2_objects_components/cuerda.png

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
The use of this new asset, along with the ripped-grass tiles from the base game, really adds some nice texture to the stealth maze. Nice work!

<img width="491" height="839" alt="Image" src="https://github.com/user-attachments/assets/bcd34f33-a729-4d3a-b09d-9398fdab9479" />

I guess this asset was also created by you, @OutsiderChikito?